### PR TITLE
Add retry logic for Selenium server download

### PR DESCRIPTION
**Pull request by vsafonkin**
Issue: [#384](https://github.com/actions/virtual-environments-internal/issues/384) 

Replaced `Invoke-WebRequest` function to helper's function `Start-DownloadWithRetry` for download Selenium server.

### CODE REVIEW

File: images/win/scripts/Installers/Install-Selenium.ps1 [hunk 1]
```diff
@@ -21,15 +23,11 @@ Write-Host "Downloading selenium-server-standalone v$seleniumVersion..."
 
 $seleniumReleaseUrl = "https://selenium-release.storage.googleapis.com/$($seleniumVersion.ToString(2))/selenium-server-standalone-$($seleniumVersion.ToString(3)).jar"
 New-Item -ItemType directory -Path "C:\selenium\"
```
Comment (al-cheb) [inline images/win/scripts/Installers/Install-Selenium.ps1:25]: nitpick: Duplicate the same variables name:
New-Item -ItemType directory -Path "C:\selenium\"
Start-DownloadWithRetry -Url $seleniumReleaseUrl -Name "selenium-server-standalone.jar" -DownloadPath "C:\selenium\"
$seleniumBinPath = "C:\selenium\selenium-server-standalone.jar"

C:\selenium - 3
selenium-server-standalone.jar - 2

Comment (vsafonkin) [reply images/win/scripts/Installers/Install-Selenium.ps1]: Refactored

File: images/win/scripts/Installers/Install-Selenium.ps1 [hunk 2]
```diff
@@ -20,16 +22,15 @@ $seleniumVersion = [version]::Parse($seleniumVersionString)
...
+
+New-Item -ItemType directory -Path $seleniumDirectory
+
+Start-DownloadWithRetry -Url $seleniumReleaseUrl -Name $seleniumFileName -DownloadPath $seleniumDirectory
 
 Write-Host "Add selenium jar to the environment variables..."
+$seleniumBinPath = Join-Path $seleniumDirectory $seleniumFileName
 setx "SELENIUM_JAR_PATH" "$($seleniumBinPath)" /M
 
 exit 0
```
Comment (vmapetr) [inline images/win/scripts/Installers/Install-Selenium.ps1:36]: I believe we can safely remove `exit 0` here. It's not really required, considering that successful provisioner execution will return 0 anyway.

**Comment by vmapetr** [comment]
Perhaps, we should also try some retires to following line
```$latestReleaseInfo = Invoke-RestMethod -Uri $latestReleaseUrl```
to make provisioner more stable.

File: images/win/scripts/Installers/Install-Selenium.ps1 (hunk 2 above)
Comment (vsafonkin) [reply images/win/scripts/Installers/Install-Selenium.ps1]: Fixed, thanks!

**Comment by vsafonkin** [comment]
> Perhaps, we should also try some retires to following line
> `$latestReleaseInfo = Invoke-RestMethod -Uri $latestReleaseUrl`
> to make provisioner more stable.

Done.

**Pull request merged**